File: library/src/main/java/com/stealthcopter/networktools/ping/PingOptions.java
package com.stealthcopter.networktools.ping;

public class PingOptions {
    private int timeoutMillis;
    private int timeToLive;

     public PingOptions() {
         timeToLive = 128;
         timeoutMillis = 1000;
     }

    public int getTimeoutMillis() {
        return timeoutMillis;
    }

    public void setTimeoutMillis(int timeoutMillis) {
        this.timeoutMillis = Math.max(timeoutMillis, 1000);
    }

    public int getTimeToLive() {
        return timeToLive;
    }

    public void setTimeToLive(int timeToLive) {
        this.timeToLive = Math.max(timeToLive, 1);
    }
}

File: library/src/main/java/com/stealthcopter/networktools/ping/PingOptions.kt
package com.stealthcopter.networktools.ping

class PingOptions {
    private var timeoutMillis = 1000
    private var timeToLive = 128
    fun getTimeoutMillis(): Int {
        return timeoutMillis
    }

    fun setTimeoutMillis(timeoutMillis: Int) {
        this.timeoutMillis = Math.max(timeoutMillis, 1000)
    }

    fun getTimeToLive(): Int {
        return timeToLive
    }

    fun setTimeToLive(timeToLive: Int) {
        this.timeToLive = Math.max(timeToLive, 1)
    }
}